### PR TITLE
feature/tools-separator

### LINKED ISSUE #9: Tools separator
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
Create tools separator.

**Describe alternatives you've considered**
No.

**Additional context**
Mobile and Desktop:
image

Link to figma:
https://www.figma.com/file/gL5DC1W2R9zhfCRRXAjJGn/CollabWorld.training?node-id=183%3A5

### PR DESCRIPTION
**Pull request by LuizGMiranda**
# What's the issue linked?

Resolve #9 

# Did you test this feature on all browsers?

Yes, tested with all the following browsers:

- [x] Firefox
- [x] Chrome

# Description

Added a new style component to the <hr /> tag to make the line, being able to pass the width and align properties.

### CODE REVIEW

**Comment by marcobrunodev** [comment]
Great!

It is important to add the component into a page I recommend you use the Dashboard page. Also, I think that you forgot to make the style.

File: src/components/RulerCollab/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const RuleCollab = ({ width, align }) => (
+  <hr noshade="noshade" width={width} align={align} size="2" />
+);
+
+RuleCollab.defaultProps = {
+  width: "100%",
+  align: "left"
+};
```
Comment (marcobrunodev) [inline src/components/RulerCollab/index.js:10]: When we use properties to change the style directly we are mixing the layers (CSS and HTML) and this is also a lack of a standard that will make the maintenance harder in the future.

**Comment by LuizGMiranda** [comment]
Add component in Dashboard page. It is not necessary make the style.

**Review by marcobrunodev — CHANGES_REQUESTED** [review]
> Add component on the Dashboard page. It is not necessary to make the style.
You created the style using the properties:
```javascript
RuleCollab.defaultProps = {
  width: "100%",
  align: "left"
};
```
It's a little bad because it leaves the maintenance more complicated. Today the standard is we have all styles of the component into the style.js file. Imagine that other developer to need to use this component and also to need to change width, this person will make by CSS into the context that the component is being used, unfortunately not will work because of the property have more value of the specificity, so her will have that to make a refactoring on component.

Please. Create a styles.js file and add the CSS's component to him.

File: src/components/RulerCollab/index.js (hunk 1 above)
Comment (LuizGMiranda) [reply src/components/RulerCollab/index.js:10]: I will create the file style.js passing the 'props', what do you think?

**Comment by LuizGMiranda** [comment]
I will create the file style.js passing the 'props', what do you think? @MarcoBrunoBR

**Pull request closed without merging**